tests/labhub_test.py

```diff
@@ -53,6 +53,14 @@ def test_invite_cmd(self):
 
         self.assertEqual(self.labhub.TEAMS, self.teams)
 
+        testbot.assertCommand(
+            '!invite meet to newcomers',
+            'ERROR: The above command cannot '
+            'be operated without nick')
+        mock_msg = create_autospec(Message)
+        mock_msg.frm.nick = PropertyMock()
+        mock_msg.frm.nick = 'batman'
+
```

Comment:
CI is failing because you are testing it wrong. We need to pass the mocked message into the bot but that is not possible at the moment while testing.
I suggest you create a class method to check if `nick` is present or not and call that for the required bot commands so that it'd be easier to test. Like this: https://github.com/coala/corobo/blob/b9d569d1461d5ffe11b8f04766221b71b8e490f9/plugins/labhub.py#L112
 Write the tests the same way I did for 'is_room_member' method.